internal/auth/yaml_storage.go

```diff
@@ -93,6 +96,10 @@ func (y *YamlStorage) RemoveCalendarAuth(calendarID string) error {
 
 func (y *YamlStorage) writeFile(cals []CalendarAuth) error {
 	var writer io.Writer
+	if strings.HasPrefix(y.StoragePath, "~/") {
+		usr, _ := user.Current()
+		y.StoragePath = filepath.Join(usr.HomeDir, y.StoragePath[2:])
```

Comment:
The path expansion must happen in the `Setup` method.

Reply:
Thank you for this comment. While debugging the error came up in the writeFile method so I added it there. It makes a lot more sense for it to happen in the Setup method and have now moved it there